app/install/macos.md

```diff
@@ -19,39 +19,52 @@ breadcrumbs:
 
     Use the [Homebrew](https://brew.sh/) package manager to add Kong as a tap and install it:
 
-    ```
+    ```bash
     $ brew tap kong/kong
     $ brew install kong
     ```
 
-2. **Prepare your database**
+1. **Add `kong.conf`**
+
+    **Note**: This step is **required** if you are using Cassandra; it is **optional** for Postgres users.
```

Comment:
No, this step is **never** required. One can create their own configuration file and store in anywhere. One can also rely on environment variables to configure Kong.

Reply:
Thank you! Am I understanding correctly that:

- A change to _either_ the config file or some environment variables is required for Cassandra 
- You can point Kong to a config file wherever you want

My goal with this was to keep the Homebrew (and source) installs as simple as possible. Would you rather make this a bit more complicated with inline instructions laying out the options, or leave it as “do this for Cassandra” and add a note at the end explaining the other options? My preference is for the latter option.

Reply:
Oh, I replied from a different UI, which didn’t add my comment to this specific thread.

Reply:
Thanks! Will update PR. Agreed that the homebrew recipe change should be a separate issue, but happy to look into that one as well.

Reply:
Further updates. Please review when you get a chance, @thibaultcha...